### DISCUSSION TITLE
Wanted help to understand how I can effectively use LabMl

### DISCUSSION THREAD

**Discussion by Aniruddha120** [labmlai/annotated_deep_learning_paper_implementations#279] [General]
Hi,
Today, I ran a notebook of labml, a multi-headed attention transformer. I just ran the cells, but I couldn't understand what is happening. what to do? I have previous experience on deep learning and machine learning, did some projects on computer vision and prediction. 

Can you please help me how to effectively use the website, so that I can understand and implement the papers? are they too much difficult for me at this stage? I hope this question suits well in this label. 

Thank you

**Comment by vpj** [comment]
Try with simpler/earlier papers first
